src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.WebKey/Properties/AssemblyInfo.cs

```diff
@@ -9,7 +9,7 @@
 [assembly: AssemblyDescription("Microsoft Azure Key Vault WebKey Class Library.")]
 
 [assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("2.0.5.0")]
+[assembly: AssemblyFileVersion("2.0.6.0")]
```

Comment:
@rijethma please delete SDKs/KeyVault/KVLocalFeed directory if it's no longer being used by any of your projects.
This was added during migration, but seems no one is using this nuget package.